src/vma/event/event_handler_manager.cpp

```diff
@@ -88,7 +88,7 @@ void* event_handler_manager::register_timer_event(int timeout_msec, timer_handle
 	void* node = malloc(sizeof(struct timer_node_t)); 
 	BULLSEYE_EXCLUDE_BLOCK_START
 	if (!node) {
-		evh_logdbg("malloc failure");
+		evh_logdbg("%s : malloc failure", __func__);
```

Comment:
VMA has a few macro's which beautify the __func__ usage in the logging.
see the 'evh_logdbg_entry()' which can be used in the function entry point